docs/directory.json

```diff
@@ -127,10 +123,6 @@
         {
           "title": "部署 EMQX 集群",
           "path": "getting-started/getting-started"
-        },
-        {
-          "title": "部署 EMQX 插件",
-          "path": "getting-started/install-emqx-prometheus-plugin"
```

Comment:
Do we need delete Markdown file ?

Reply:
> Do we need delete Markdown file ?

Xuebin and I confirmed that this document can be deleted. We have a document that specifically introduces the use of prometheus to collect EMQX indicators, so this document is meaningless.